WebApplication/Processing/ProjectWork.cs

```diff
@@ -172,22 +173,30 @@ public async Task<FdaStatsDTO> GenerateRfaAsync(string projectName, string hash)
             return FdaStatsDTO.All(result.Stats);
         }
 
-        public async Task<FdaStatsDTO> ExportDrawingPdfAsync(string projectName, string hash)
+        public async Task<(FdaStatsDTO, int)> ExportDrawingPdfAsync(string projectName, string hash, string drawingKey)
         {
             _logger.LogInformation($"Getting drawing pdf for hash {hash}");
 
             ProjectStorage storage = await _userResolver.GetProjectStorageAsync(projectName);
             Project project = storage.Project;
 
             var ossNames = project.OssNameProvider(hash);
+            var ossAttributes = project.OssAttributes;
 
+            // get drawing index from drawing specified
             var bucket = await _userResolver.GetBucketAsync();
+
+            // ????? cache drawingsList instead of reading it every time ?????
```

Comment:
I think it's already cached here:
https://github.com/Developer-Autodesk/forge-configurator-inventor/blob/46d5b51514a5e1cbfcd628357c756a2ae408c576/WebApplication/State/ProjectStorage.cs#L97

Reply:
ok, and how is the right way to get content, when it is cached? thank you!

Reply:
Should be similar to this:
https://github.com/Developer-Autodesk/forge-configurator-inventor/blob/4c36d4d963e35252fad76d7de60f8f9b6e00cf8a/WebApplication/Processing/ProjectWork.cs#L133